functions/backup.bash

```diff
@@ -52,23 +52,25 @@ restore_openhab_config() {
 
   echo -n "$(timestamp) [openHABian] Choosing openHAB backup zipfile to restore from... "
   if [[ -n "$INTERACTIVE" ]]; then
-    # shellcheck disable=SC2012
-    readarray -t backupList < <(ls -alh "${backupPath}"/openhab*-backup-* 2> /dev/null | head -20 | awk -F ' ' '{ print $9 " " $5 }' | xargs -d '\n' -L1 basename | awk -F ' ' '{ print $1 "\n" $1 " " $2 }')
-    if [[ -z "${backupList[*]}" ]]; then
-      whiptail --title "Could not find backup!" --msgbox "We could not find any configuration backup file in the storage directory $backupPath" 8 80
-      echo "CANCELED"
-      return 0
+    if [[ $# -gt 0 ]]; then
+      if ! [[ -s "$1" ]]; then echo "FAILED (restore config $1)"; return 1; fi
```

Comment:
added whiptail feedback
Not sure about your feedback regarding textonly: this PR did not change anything about it?
The intended logic is that *if* a user knows what he is to import is text only then he can choose the secondary (recently added) menu option that calls the restore script with --textonly because that option will NOT restart OH.
User can also select the primary option then it gets restarted nonetheless (as is needed when some non-text config is like jsondb and UI's json are imported). Note this is independent of what is contained in the zipfile to restore.
I'd hope this logic is what I implemented, please validate.